frontend/src/Components/ProductListingPage/MobileSortAndFilter.jsx

```diff
@@ -0,0 +1,134 @@
+
+import { useProduct } from "../../Context/ProductContext";
+
+import "./MobileSortAndFilter.css";
+import "./FilterSideBar.css"
+const sortByNames = ["Latest","Discount","Price : High to Low","Price : Low to High"]
```

Comment:
Could be moved into utils to avoid code repetition.

Reply:
done